test/integration/scheduler/scheduler_test.go

```diff
@@ -1291,6 +1301,34 @@ var _ = ginkgo.Describe("Scheduler", func() {
 			util.ExpectWorkloadsToBePending(ctx, k8sClient, wl2)
 			util.ExpectPendingWorkloadsMetric(strictFIFOClusterQ, 0, 1)
 		})
+
+		ginkgo.FIt("Pending workload with StrictFIFO doesn't block other CQ from borrowing", func() {
+			ginkgo.By("Creating ClusterQueues and LocalQueues")
+			lqTeamA := testing.MakeLocalQueue("team-a-lq", matchingNS.Name).ClusterQueue(cqTeamA.Name).Obj()
+			gomega.Expect(k8sClient.Create(ctx, lqTeamA)).Should(gomega.Succeed())
+
+			ginkgo.By("Creating workloads")
+			wl1 := testing.MakeWorkload("wl", matchingNS.Name).Queue(strictFIFOLocalQueue.
+				Name).Request(corev1.ResourceCPU, "5").Priority(10).Obj()
+			gomega.Expect(k8sClient.Create(ctx, wl1)).Should(gomega.Succeed())
+
+			wl2 := testing.MakeWorkload("player1-a", matchingNS.Name).Queue(lqTeamA.
+				Name).Request(corev1.ResourceCPU, "5").Priority(1).Obj()
+			gomega.Expect(k8sClient.Create(ctx, wl2)).Should(gomega.Succeed())
+
```

Comment:
Move `util.ExpectWorkloadsToBeAdmitted(ctx, k8sClient, wl1, wl2)` here.

This is to be 100% that the workloads are admitted before we create another workload, which could potentially move ahead of the others if the timestamps end up being the same, because timestamps only have granularity by seconds.